inlong-common/src/main/java/org/apache/inlong/common/reporpter/AbstractReporter.java

```diff
@@ -101,14 +101,15 @@ public Response syncReportData(T data, String serverUrl) throws Exception {
             return null;
         }
         HttpPost httpPost = new HttpPost(serverUrl);
+        String returnStr = null;
         try {
             StringEntity stringEntity = new StringEntity(gson.toJson(data));
             stringEntity.setContentType(AGENT_HTTP_APPLICATION_JSON);
             httpPost.setEntity(stringEntity);
-            String returnStr = executeHttpPost(httpPost);
+            returnStr = executeHttpPost(httpPost);
             return parse(returnStr);
         } catch (Exception e) {
-            LOGGER.error("syncReportData has exception e = {}", e);
+            LOGGER.error("syncReportData has exception returnStr = {}, e:", returnStr, e);
             throw e;
```

Comment:
It is better to return a Response with error code.

Reply:
Response protocol do not support error code felid now. Error code need be add later.